docs/getting-started/user-roadmap-apiml.md

```diff
@@ -0,0 +1,165 @@
+# Information roadmap for Zowe API Mediation Layer
+
+This roadmap outlines information resources that are applicable to the various user roles who are interested in Zowe API Mediation Layer. These resources provide information about various subject areas, such as learning basic skills, installation, developing, and troubleshooting for Zowe API Mediation Layer.
+
+The following definition of skill levels about Zowe assist you with gathering the most relevant resources for you. 
+
+* Beginner: You're starting out and want to learn the fundamentals.
+* Intermediate: You have some experience but want to learn more in-depth skills. 
+* Advanced: You have lots of experience and are looking to learn about specialized topics.
+
+## Fundamentals
+
+> Zowe skill level: Beginner
+
+- [**Zowe API Mediation Layer overview**](overview.md#api-mediation-layer)
+
+   New to API Mediation Layer? This overview topic introduces the key features, main components, benefits, and architecture of API Mediation Layer.
+
+- [**Architecture**](zowe-architecture.md#zowe-architecture)
+
+   Review the Zowe architecture to understand how API Mediation Layer works in the Zowe framework.
+
+## Installing
+
+> Zowe skill level: Beginner
+
+- [**System requirements**](../user-guide/systemrequirements.md)
+
+   Review this topic to ensure that your system meets the requirements for installing the API Mediation Layer. API Mediation Layer is one of the server-side components. 
+
+- [**Planning**](../user-guide/installandconfig.md#planning-the-installation-of-zowe-server-components)
+
+  This article includes details about planning for installation, the Zowe z/OS launch process, and information about the Zowe runtime directory, instance directory, and keystore directory.
+
+- [**Installing API Mediation Layer**](../user-guide/install-zos.md#z-os-installation-roadmap)
+
+   This article provides an overview of the essential steps involved in installing API Mediation Layer.
+
+## Configuring and updating
+
+> Zowe skill level: Intermediate
+
+- **Configuring API Mediation Layer**
+
+   - [API Gateway configuration parameters](../user-guide/api-mediation/api-mediation-internal-configuration.md)
+     
+      This article introduces the default configuration of the API Mediation Layer and how to change the parameters to meet your needs. 
+
+   - [Configuring the Zowe APIs](../user-guide/configure-data-sets-jobs-api.md) 
+
+      The article explains how to configure security for the Zowe API Mediation Layer. 
+
+   - [Advanced Gateway features configuration](../user-guide/api-mediation/api-gateway-configuration.md)
+      
+      This article is for system programmers who want to configure advanced Gateway features of the API Mediation Layer, such as the gateway retry policy, connection limits, gateway timeouts, and so on.
+
+- [**Upgrading the z/OS system for Zowe**](../user-guide/upgrade-zos-system.md)
+
+   Learn how to upgrade Zowe API Mediation Layer to a recent version.
```

Comment:
Why it's names Upgrading the zOS system but here you say upgrade API ML to a recent version?